cogs/info.py

```diff
@@ -1,11 +1,13 @@
 from asyncio import wait_for
+
+import discord
 from discord import slash_command, Embed
-from discord.ext import commands
 from discord.ext.commands import Bot
+
 from utilities.data import Colors, get_server_status
```

Comment:
You know the deal

Reply:
Done to seperate third party imports (discord) from local imports (utilities.data) and built-in imports (asyncio)